pkg/plugins/runtime/gateway/testdata/http/cross-mesh-gateway.yaml

```diff
@@ -729,9 +729,9 @@ Routes:
               clusters:
               - name: external-http2-httpbin-29c391043cf34cd7
                 weight: 1
-    edge-gateway:HTTP:8081:
+    edge-gateway:HTTP:8081:*:
       ignorePortInHostMatching: true
-      name: edge-gateway:HTTP:8081
+      name: edge-gateway:HTTP:8081:*
```

Comment:
There's no size limit on names?

Reply:
I was not able to find any

Reply:
Should we add a maxlength to listener.hostname?

Reply:
Let's do it

Reply:
@lahabana @jakubdyszkiewicz done, added to UPGRADE.md